workflows/update_tags.sh

```diff
@@ -25,28 +25,35 @@
 # This is necessary since a new docs-csm version could land before or after a CSM update.
 
 set -e
-REGISTRY="registry.local"
+THIS_REGISTRY_NAME="registry.local"
+THIS_REGISTRY_PROTOCOL="https"
+THIS_PODMAN_TLS=""
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 cd $SCRIPT_DIR
 
+
+# Ensure the script is running from within the workflows directory so it finds argo yaml files."
 if [[ "$(basename ${SCRIPT_DIR})" != "workflows" ]]; then
   echo "ERROR: This script must be located in the workflows directory to run."
   exit 1
 fi
 
-if [[ $(nslookup $REGISTRY >> /dev/null; echo $?) != 0 ]]; then
-  echo "INFO: Skipping update_tags.sh script given docs were installed outside of a CSM environment."
-  exit 0
-fi
+# Change registry URL target for PIT
+function update_registry_url_to_pit() {
+  echo "INFO: Running on PIT. Updating registry URL."
+  export THIS_REGISTRY_NAME="pit.nmn:5000"
+  export THIS_REGISTRY_PROTOCOL="http"
+  export THIS_PODMAN_TLS="--tls-verify=false"
+}
 
-if [[ -f /etc/pit-release ]]; then
-  echo "INFO: Skipping update_tags.sh script given docs were installed on PIT."
-  exit 0
-fi
+[[ ! $(curl -s ${THIS_REGISTRY_PROTOCOL}://${THIS_REGISTRY_NAME}) ]] && update_registry_url_to_pit
 
-if [[ ! $(curl -s https://${REGISTRY}) ]]; then
-  echo "ERROR: Is this a CSM environment? Check that ${REGISTRY} is accessible over https before continuing."
-  exit 1
+if [[ ! $(curl -s ${THIS_REGISTRY_PROTOCOL}://${THIS_REGISTRY_NAME}) ]]; then
+  echo "WARNING: Unable to update workflow image tag references. ${THIS_REGISTRY_NAME} is not accessible."
```

Comment:
This may be confusing because it will always say "pit.nmn:5000 is not accessible".   That may lead one to think that they need to get PIT nexus up it will likely usually be registry.local that needs to be accessible.

Reply:
Changed to "the registry is not accessible".